src/components/Home.js

```diff
@@ -3,9 +3,22 @@ import { Button } from './Button';
 import { SearchInput } from './SearchInput';
 import { CategoriesList } from './CategoriesList';
 
+const randomNumber = (max, min) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+let classNumber;
+
+const headerClass = () => {
+  if (!classNumber){
+    classNumber = randomNumber(0 , 7);
+  }
```

Comment:
The random images seem to be working, but it looks like this is backwards. Shouldn't be it randomNumber(7, 0)?